src/e3/testsuite/driver/adacore.py

```diff
@@ -128,14 +128,16 @@ def get_script_command_line(self) -> List[str]:
             )
 
         _, ext = os.path.splitext(self.script_file)
+
+        is_cmd = ext.startswith(".cmd")
```

Comment:
The rest of the code checks for equality with `ext`: what’s the reason here to use `.startswith`?

Reply:
The fixedbugs testsuite has a few tests whose extension is ".cmd_something" (e.g. test.cmd_x86, test.cmd_zfp...). In one case checking for `ext == ".cmd"` rather than `ext.startswith(".cmd")` caused a failure.

Reply:
Ah interesting, thanks! Could you add a comment that describes this case? Adding a testcase for it would be nice, too!